src/els_uri.erl

```diff
@@ -69,7 +69,7 @@ uri(Path) ->
   els_utils:to_binary(
     uri_string:recompose(#{
       scheme => <<"file">>,
-      host => http_uri:encode(Host),
+      host => Host,
```

Comment:
My reading of [uri_string:recompose/1](https://erlang.org/doc/man/uri_string.html#recompose-1) is that *it* will do all necessary encoding, we do not need to do it for the sub-components.

Reply:
In my branch (I was doing the same), I just stripped it away.